src/draw/handler/Draw.Circle.js

```diff
@@ -15,7 +15,8 @@ L.Draw.Circle = L.Draw.SimpleShape.extend({
 			clickable: true
 		},
 		showRadius: true,
-		metric: true // Whether to use the metric meaurement system or imperial
+		metric: true, // Whether to use the metric meaurement system or imperial
+		toolbarIcon: { className: 'leaflet-draw-draw-circle', tooltip: 'Draw a circle.' }
```

Comment:
Leaflet.draw uses a bit of a hack to support localization. See https://github.com/Leaflet/Leaflet.draw/blob/master/src/Leaflet.draw.js So for the tooltip text you would use `Leaflet.draw.circle.tooltip.start`.